Automattic/wc-calypso-bridge
# Replace product task with custom completion logic

### Changes proposed in this Pull Request:

Closes https://github.com/Automattic/wc-calypso-bridge/issues/955.

This PR adds a replacement product class that uses a custom completion logic. Applies to all plans.

However, this PR does not address if the user modifies the sample product, it'll simply not be counted as a published product.



### How to test the changes in this Pull Request:

1. Make sure there are no products in your site
2. Import products [CSV from headstart](https://gist.githubusercontent.com/ilyasfoo/1907a9d5e5b8a72fefe2af6e8d05ef03/raw/c3f3e5e2a52169480b9aeab6e1ec8e672fbb73db/headstart-product-export.csv) 
3. Go to WooCommerce > Home, observe that product task is not marked as completed
5. Add a new product and publish it
6. Go to WooCommerce > Home, observe that product task is marked as completed
